verifier/verifier.go

```diff
@@ -554,8 +554,8 @@ func logVerificationResult(logger log.Logger, result *notation.ValidationResult)
 	}
 	switch result.Action {
 	case trustpolicy.ActionLog:
-		logger.Warnf("%v validation failed with validation action set to \"logged\". Failure reason: %v", result.Type, result.Error)
+		logger.Warnf("%v validation failed with validation action set to \"logged\". Failure reason: %v", strings.Title(string(result.Type)), result.Error)
 	case trustpolicy.ActionEnforce:
-		logger.Errorf("%v validation failed. Failure reason: %v", result.Type, result.Error)
+		logger.Errorf("%v validation failed. Failure reason: %v", strings.Title(string(result.Type)), result.Error)
```

Comment:
We should just use `result.Type` as it is.

Reply:
If we want it capitalized, think about `Validation %v` instead of `%v validation`.